tests/test_lightningd.py

```diff
@@ -1439,17 +1454,16 @@ def test_closing_different_fees(self):
             if p.amount != 0:
                 self.pay(l1, p, 100000000)
 
-        # Now close
+        # Now close all channels
         # All closes occur in parallel, and on Travis,
         # ALL those lightningd are running on a single core,
         # so increase the timeout so that this test will pass
         # when valgrind is enabled.
-        # (close timeout defaults to 30 as of this writing, more
-        # than double the default)
-        closes = [self.executor.submit(l1.rpc.close, p.info['id'], False, 72) for p in peers]
+        # (close timeout defaults to 30 as of this writing)
+        closes = [self.executor.submit(l1.rpc.close, p.channel, False, 90) for p in peers]
 
         for c in closes:
-            c.result(72)
+            c.result(90)
```

Comment:
Looking at the logs on Travis this doesn't seem to be a timeout issue at all, there's more than a minute of radio-silence, no progress on fee negotiation.

Reply:
You may be right; there may be a heisenbug here when closing multiple channels at the same time. In one run I am looking at now, all 6 peers transitioned their channel to `CLOSINGD_COMPLETE`, but `l1` only transitioned 5 channels to `CLOSINGD_COMPLETE`, so something is broken somewhere.